atmos_model.F90

```diff
@@ -2490,6 +2491,15 @@ subroutine assign_importdata(jdat, rc)
               GFS_data(nb)%Coupling%hsnoin_cpl(ix) = min(hsmax, GFS_data(nb)%Coupling%hsnoin_cpl(ix) &
                              / (GFS_data(nb)%Sfcprop%fice(ix)*GFS_data(nb)%Sfcprop%oceanfrac(ix)))
               GFS_data(nb)%Sfcprop%zorli(ix)       = z0ice
+              tem = GFS_data(nb)%Sfcprop%tisfc(ix) * GFS_data(nb)%Sfcprop%tisfc(ix)
+              tem = con_sbc * tem * tem
```

Comment:
Just a suggestion, this can (safely) be written as `tem = con_sbc * GFS_data(nb)%Sfcprop%tisfc(ix)**4`. Gone are the days when this was necessary. Compilers know how to optimize these kind of expressions.